sentry/sentry.conf.example.py

```diff
@@ -134,6 +134,9 @@ def get_internal_network():
 
 KAFKA_CLUSTERS["default"] = DEFAULT_KAFKA_OPTIONS
 
+# Temporary flag to test a new ingest consumer for user feedback.
+SENTRY_OPTIONS["feedback.ingest-topic.rollout-rate"] = 1.0
```

Comment:
Will the default value of this option be changed in sentry? How will we know it is time to remove this temporary flag?

Reply:
The default value is 0.0 and will not be changed

Reply:
As soon as this ingest topic is tested and confirmed to work in self-hosted, I will remove references to the flag in all repos.

Reply:
Got it, so essentially the option will not be needed in the future after the references to the flag is removed

Reply:
Yup!